tldr.py

```diff
@@ -305,17 +305,24 @@ def get_page(
 PARAM_REGEX = re.compile(r'(?:{{)(?P<param>.+?)(?:}})')
 
 
-def get_commands(platforms: Optional[List[str]] = None) -> List[str]:
+def get_commands(platforms: Optional[List[str]] = None, language: Optional[str] = None) -> List[str]:
     if platforms is None:
         platforms = get_platform_list()
 
+    if language:
+        languages = [get_language_code(language[0])]
+    else:
+        languages = get_language_list()
+
     commands = []
     if get_cache_dir().exists():
         for platform in platforms:
-            path = get_cache_dir() / 'pages' / platform
-            if not path.exists():
-                continue
-            commands += [file.stem for file in path.iterdir() if file.suffix == '.md']
+            for language in languages:
+                pages_dir = f'pages.{language}' if language != 'en' else 'pages'
+                path = get_cache_dir() / pages_dir / platform
+                if not path.exists():
+                    continue
+                commands += [file.stem for file in path.iterdir() if file.suffix == '.md']
```

Comment:
```suggestion
                commands += ["file.stem ({language})" for file in path.iterdir() if file.suffix == '.md']
```

If we do want to search for pages in all languages, we should show the language of each page.

Reply:
Good idea but your suggestion does not make the string an f-string. Could you please update it?

Reply:
@frenzymadness The suggestion has been updated (feel free to apply it).